manifests/peer-validation.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kilo
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "peer-validation.kilo.svc"
+webhooks:
+- name: "peer-validation.kilo.svc"
```

Comment:
I think that this name is normally namespaced to the domain of the project rather than the DNS name of a service within the project. In our case it could be `peers.kilo.squat.ai`, e.g. https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/config/webhook/manifests.yaml#L18